public/mobile-app/src/routes/+page.svelte

```diff
@@ -47,30 +73,40 @@ const retrieveNotifications = async () => {
 const checkNotificationPermission = async () => {
   // Check if the browser supports notifications
   if (!('Notification' in window)) {
-    alert('Add this website to your home screen!')
+    console.error('Add this website to your home screen!')
     return false
   }
   const permission = await Notification.permission
   console.log('permission:', permission)
   return permission == 'granted'
 }
 
+const resetElements = () => {
+  isAuthenticatedForNotifications = false
+  authenticationStatus = ''
+  isRegisteredWithAmi = false
+  registrationStatus = ''
+  pushSubscription = null
+  registerEmailInputValue = ''
+  userEmailState = ''
+  userNotifications = []
+  window.localStorage.setItem('emailLocalStorage', '')
+  window.localStorage.setItem('pushSubscriptionLocalStorage', '')
+}
```

Comment:
I wonder if we could use svelte runes (`$state` and `$derived`) to automatically unset some of those?
We might want to make sure it doesn't make the code too difficult to understand and maintain. We wouldn't want to increase the complexity for no good reason.